lib/src/common/frame.dart

```diff
@@ -0,0 +1,23 @@
+part of tiled;
+
+/// <frame>
+///
+/// * tileid: The local ID of a tile within the parent <tileset>.
+/// * duration: How long (in milliseconds) this frame should be displayed
+///   before advancing to the next frame.
```

Comment:
Use doc refs here

Reply:
these actually refer to the xml tags, that are always lowercase, so not sure if the ref would work for all elements

Reply:
I think it makes sense to have the tags there, I also misunderstood what they were in my comment.

Reply:
I will add a text to clarify what this is on eveyr file